common/src/test/java/org/apache/sedona/common/FunctionsTest.java

```diff
@@ -268,6 +268,38 @@ public void splitHeterogeneousGeometryCollection() {
         assertNull(actualResult);
     }
 
```

Comment:
Please separate different testCases to different tests since these are unit tests.

You can have tests like dimensionGeom2D, dimensionGeomCollection, dimensionGeomEmpty, etc.

Also, use assertEquals(expected, actual) format so that the test failure flags expected and actual properly.

Additionally, please add test cases testing 3D geometry as well